Awesome/styles2.css

```diff
@@ -0,0 +1,536 @@
+  padding: 10px 0;
+  border: 0px;
+}
+
+.awesome__tab-checked{
+  width:200px;
+  text-transform: uppercase;
+  font-family: 'Open Sans', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 19px;
+  background-color: rgb(255, 255, 255);
+  padding: 10px 0;
+  border: 1px solid #C6CBCD;
+  border-bottom: none;
+  border-radius: 3px;
+  margin-bottom:-2px;
+}
+
+.informaton{
+  margin: 0 auto;
+  margin-top: 100px;
+  width: 1170px;
+  
+}
+
+.awesome__tabs-container{
+  display: flex;
+  flex-direction: row;
+  justify-content: left;
+}
+
+.awesome__tab-content{
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+  margin-bottom: 50px;
+  border: 1px solid #C6CBCD;
+  border-radius: 1px;
+}
+
+.awesome__tab-content p{
+  font-family: Raleway;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 18px;
+  line-height: 21px;
+  margin-right: 30px;
+  padding-left: 100px;
+}
+
+.awesome__tab-content h2{
+  font-family: 'Open Sans';
+  font-style: normal;
+  font-weight: bold;
+  font-size: 30px;
+  line-height: 41px;
+  padding: 40px 100px 0;
+  margin-bottom: 25px;
+}
+
+.awesome__tab-content button{
+  font-family: 'Open Sans';
+  font-style: normal;
+  font-weight: 600;
+  font-size: 18px;
+  color:white;
+  line-height: 25px;
+  text-align: center;
+  text-transform: capitalize;
+  width: 270px;
+  height: 58px;
+  margin: 28px 100px;
+  background: #EC7979;
+  border: none;
+}
+
+/*Section Pricing options styles*/
+
+.pricing{
+  margin-top: 60px;
+  padding-bottom: 50px;
+  background: #F5F5F5;
+  width: 100%;
+}
+
+.pricing__tittle-wrapper{
+  padding-top: 50px;
+  height: 150px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.pricing__sheet{
+  margin-top: 80px;
+  display: flex;
+  justify-content: center;;
+}
+
+.pricing__card{
+  width: 370px;
+  margin: 0 30px;
+}
+
+.pricing__card_hover{
+  width: 370px;
+  margin-top: -20px;
+}
+
+.card__shadow{
+  box-shadow: 0px 0px 10px 2px #b4b4b4;
+}
+
+.pricing__card-top{
+  background: #333333;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.pricing__card-title{
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 60px;
+  font-family: "Open Sans";
+  font-style: normal;
+  font-weight: bold;
+  font-size: 24px;
+  line-height: 33px;
+  text-transform: capitalize;
+  color: #FFFFFF;
+  margin: auto;
+  vertical-align: middle;
+}
+
+.pricing__card-price{
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 100px;
+  font-family: Raleway;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 36px;
+  line-height: 42px;
+  color: #FFFFFF;
+  border: 2px solid #F5F5F5;
+  border-right: none;
+  border-left: none;
+}
+
+.pricing__card-subtitle{
+  height: 50px;
+  font-family: "Raleway";
+  font-style: normal;
```

Comment:
You don't need to add font-style everywhere.

Reply:
Deleted font-style: normal;. Is it a default value?